hhub/views.py

```diff
@@ -108,3 +121,24 @@ def search_entries(request):
 
 def entry_platform(request, platform):
     return
+
+#########
+# UTILS #
+#########
+def sort_and_order(entries, order_by, sort_by):
+    try:
+        # order_by a specific field from schema
+        if order_by != 1 and order_by != "":
+            # specify an order --> asc is by default, so we're not dealing with it
```

Comment:
Not clear: if we're *not* dealing with it, why in 2 lines we are dealing with it?